modules/lib/maintainers.nix

```diff
@@ -418,4 +418,10 @@
     github = "liyangau";
     githubId = 71299093;
   };
+  hitsmaxft = {
+    name = "Bhe Hongtyu";
+    email = "mfthits@gmail.com";
+    github = "hitsmaxft";
+    githubId = 352727;
+  };
```

Comment:
You seem to already be in the Nixpkgs maintainer list so no need to add yourself here.
```suggestion
```